driver/tke_driver.go

```diff
@@ -1013,96 +1035,183 @@ func (d *Driver) GetK8SCapabilities(ctx context.Context, opts *types.DriverOptio
 	return capabilities, nil
 }
 
-func getWrapAddClusterInstancesRequest(state, newState *state, nodeCount int64) (*ccs.AddClusterInstancesRequest, error) {
-	logrus.Debugf("invoking get wrap request of AddClusterInstances")
-	// goodsNum is the new nodes count that will be added to the cluster
-	state.GoodsNum = newState.NodeCount - nodeCount
+func removeClusterInstances(state, newState *state, nodeCount int64, instancesResp *tke.DescribeClusterInstancesResponse) (*tke.DeleteClusterInstancesRequest, error) {
```

Comment:
Looks like nodeCount is from instancesResp so we don't need 4 params for this function.